fuzzing/src/main/java/fuzzing/KeysFuzzer.java

```diff
@@ -26,12 +26,20 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) {
     try {
       String[] schemes = {"rsassa-pss-sha256", "ed25519", "ecdsa-sha2-nistp256"};
       String scheme = data.pickValue(schemes);
+      Boolean choice = data.consumeBoolean();
       byte[] byteArray = data.consumeRemainingAsBytes();
 
-      Keys.parsePublicKey(byteArray);
-      Keys.constructTufPublicKey(byteArray, scheme);
+      if (choice) {
```

Comment:
What do you think of splitting in into several fuzzers?

Reply:
Thanks for the suggestion.